elements/pfe-primary-detail/src/pfe-primary-detail.scss

```diff
@@ -0,0 +1,98 @@
+// Please see the pfe-sass README for guidance on using these tools
+@import "../../pfe-sass/pfe-sass";
+
+
+$LOCAL: primary-details;
+
+// This variable is global so that helper functions can reference it
+$LOCAL-VARIABLES: (
+  Border: 1px solid pfe-var(ui--border--lighter),
+
+  // Applied to the layout of the nav vs details
+  GridTemplateColumns: 1fr 2fr,
+
+  // Styles have important since they are slotted
+  nav: (
+    Color: #151515!important, // pfe-var(text)
+    Color--active: #06c!important, // pfe-var(link)
+    Background--active: #f0f0f0!important, // pfe-var
+  ),
+  details: (
+    Background: pfe-var(surface--lightest),
+  ),
+);
+
+* {
+  box-sizing: border-box;
+}
+
+:host {
+  display: grid;
+  grid-template-columns: pfe-local(GridTemplateColumns);
+  border: pfe-local(Border);
+  align-items: stretch;
+  justify-content: stretch;
+}
+
+:host([hidden]) {
+  display: none;
+}
+
+#details-nav {
+  display: flex;
+  flex-direction: column;
+  padding: 9px 0;
+  border-right: pfe-local(Border);
+}
+
+::slotted([slot="details-nav"]) {
+  display: block!important;
+  appearance: none!important;
+  background: transparent!important;
+  border: 0!important;
+  margin: 0!important;
+  padding: 0.5em 1.5em 0.5em 0.75em!important;
+  font: inherit;
+  line-height: 1.4;
```

Comment:
Why 1.4 rather than 1.5 here?

Reply:
1.4 is the number I've always used as 'tight leading' for things like button and menu items. My default leading is 1.6/1.8 depending on the font.

I was on auto-pilot when I added it. 

Is there a preferred way of handling this? Make it a local var? Use a theme var?

Reply:
There is a theme var and I'd check the mock-ups.  Corey tends to use 1.5 from what I've seen but let me tag @starryeyez024 who has been in typography land all month

Reply:
I updated it to 1.5